pkg/operator/k8sutil/service.go

```diff
@@ -62,6 +62,11 @@ func UpdateService(
 	return clientset.CoreV1().Services(namespace).Update(serviceDefinition)
 }
 
+//DeleteService makes a best effort at deleting a Services, then waits for them to be deleted
+func DeleteService(clientset kubernetes.Interface, namespace, name string) error {
+	return clientset.CoreV1().Services(namespace).Delete(name, &metav1.DeleteOptions{})
```

Comment:
if the err is that the service is not found, let's return `nil` instead of the error.